src/test/java/org/folio/util/AssignedUsersTestUtil.java

```diff
@@ -64,14 +61,6 @@ public static String insertAssignedUser(String credentialsId, String username, S
     return future.join();
   }
 
-  public static String resourcePath(String... params) {
```

Comment:
Why do you think that this method is not useful?

Reply:
it is not clear from my perspective what is the resource path in the current test is used and the method name does not help to understand the context of  resourcePath